AiTableTopGameMaster.ConsoleApp/Helpers/ServiceExtensions.cs

```diff
@@ -26,6 +29,13 @@ public static ServiceProvider BuildServiceProvider(IAnsiConsole console, string[
         services.AddTransient<Kernel>(sp =>
         {
             IKernelBuilder builder = Kernel.CreateBuilder();
+            builder.Services.AddLogging(loggingBuilder =>
```

Comment:
Can we somehow combine or consolidate this with our existing logging extensions used by program?